### PR TITLE
docs(CONTRIBUTING.md): add some real examples of scopes

### PR DESCRIPTION
**Pull request by zetok**
What other scopes could be in examples?

It would be nice if examples could list scopes that would fit like 80-90% of use cases.

---

This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/3592)

### CODE REVIEW

**Comment by Diadlo** [comment]
Seems `CONTRIBUTING.md` longer, than dozen. And this can be an example (i.e. `CONTRIBUTING`)

**Note:** Reviewable doesn't show line breaks (`\n` replaced on spaces)

**Comment by initramfs** [comment]
Perhaps mirror the source folders (`net`, `core`, etc)? `qtox.pro` or something like `build` for build script tweaks.

**Comment by zetok** [comment]
@initramfs I'm not sure if mirroring folder names is a good idea.

I.e. in the past there already was confusion about "what to put in scope if change spans across several files", and this would just move the issue from files to directories.

`build` sounds good though, and a complimentary to it would be `travis`.

**Comment by initramfs** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3592#-:-KOQ3bGAy3hov78aTPDv:bnfp4nl)_

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -204,9 +226,9 @@ E.g. https://github.com/qTox/qTox/blob/master/src/misc/flowlayout.cpp
 
 ## Documentaion
 
-If you added a new function, also add a doxygen comment before the implementation.
-If you changed an old function, make sure the doxygen comment is still correct.
-If it doesn't exist add it.
+If you added a new function, also add a doxygen comment before the
+implementation.  If you changed an old function, make sure the doxygen comment
```
Comment (ghost) [inline CONTRIBUTING.md]: Double space

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -204,9 +226,9 @@ E.g. https://github.com/qTox/qTox/blob/master/src/misc/flowlayout.cpp
...
+is still correct.  If it doesn't exist add it.
```
Comment (ghost) [inline CONTRIBUTING.md]: Double space

**Comment by zetok** [comment]
Fixed double space.

**Comment by zetok** [comment]
> `simple_make.sh` is to long

Yep, and it shouldn't be used – instead, there is `build`

**Comment by Diadlo** [comment]
@zetok Oh.. Sorry. Alignment confused me

**Comment by Diadlo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3592#-:-KOUju8KN5mSER5DZDOw:bnfp4nl)_

**Pull request merged**